src/main/java/org/folio/circulation/domain/notice/schedule/ScheduledNoticeRepository.java

```diff
@@ -0,0 +1,47 @@
+package org.folio.circulation.domain.notice.schedule;
+
+import static org.folio.circulation.domain.notice.schedule.JsonScheduledNoticeMapper.mapFromJson;
+import static org.folio.circulation.support.Result.failed;
+
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.circulation.support.Clients;
+import org.folio.circulation.support.CollectionResourceClient;
+import org.folio.circulation.support.ForwardOnFailure;
+import org.folio.circulation.support.Result;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.vertx.core.json.JsonObject;
+
+public class ScheduledNoticeRepository {
+
+  private static final Logger log = LoggerFactory.getLogger(ScheduledNoticeRepository.class);
+
+  public static ScheduledNoticeRepository using(Clients clients) {
+    return new ScheduledNoticeRepository(
+      clients.scheduledNoticeStorageClient());
+  }
+
+  private final CollectionResourceClient scheduledNoticeStorageClient;
+
+
+  public ScheduledNoticeRepository(
```

Comment:
This can be private, also I think it reads better plural e.g. `ScheduledNoticesRepository`

```suggestion
  private ScheduledNoticesRepository(
```

Reply:
Done